improver/nbhood/__init__.py

```diff
@@ -6,6 +6,8 @@
 
 from typing import List, Optional, Tuple, Union
 
+from improver.utilities.common_input_handle import as_iterable
+
 
 def radius_by_lead_time(
     radii: List[str], lead_times: Optional[List[str]] = None
```

Comment:
```suggestion
    radii: Union[str, List[str]], lead_times: Optional[List[str]] = None
```
The doc-string needs updating too.